webapp/client/src/pages/MerkzeichenPersonAPage.test.js

```diff
@@ -0,0 +1,95 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { within } from "@testing-library/dom";
+import MerkzeichenPersonAPage from "./MerkzeichenPersonAPage";
+import { Default as StepFormDefault } from "../stories/StepForm.stories";
```

Comment:
Could we maybe rename this file and test the MerkzeichenPage directly?

Reply:
And then do not test the person A page?

Reply:
Exactly, because the component was already tested and the PersonA component does not add any functionality.